package.json

```diff
@@ -11,8 +11,8 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/parity-tech/react-native-substrate-sign.git",
-    "baseUrl": "https://github.com/github_account/react-native-substrate-sign"
+    "url": "git+https://github.com/paritytech/react-native-substrate-sign.git",
+    "baseUrl": "https://github.com/paritytecht/react-native-substrate-sign"
```

Comment:
all of a sudden I've randomly checked this URL and it's 404

Reply:
```suggestion
    "baseUrl": "https://github.com/paritytech/react-native-substrate-sign"
```

Reply:
Thanks, typo indeed.